src/components/Basket.vue

```diff
@@ -1,29 +1,35 @@
 <template>
   <div class="basket">
-    <h1 v-if="$store.state.shoppingCart.cartId === 0">
+    <h1 v-if="$store.state.shoppingCart[0].full === false">
```

Comment:
Mówiłem już, że komponent ma NIE korzystać z zmiennej globalnej $store. 
Stwórz getter do tego

Reply:
Jak zrboisz getter, podepnij je do map getters. I tak, $store.state.shoppingCart to getter